testingutils/identity/identity.go

```diff
@@ -64,7 +65,8 @@ func CreateAccountIDWithKeys(contextTimeout time.Duration, acc *configstore.Acco
 
 func GenerateRandomDID() identity.DID {
 	r := utils.RandomSlice(identity.DIDLength)
-	return identity.NewDIDFromBytes(r)
+	did, _ := identity.NewDIDFromBytes(r)
```

Comment:
return error

Reply:
I think that in most cases it makes sense to return the error, but in this function I think it creates more work than helping to debug, as we are just generating the randomDID from a specified bytelength, which leaves no room for bugs resulting from user input. Returning the error here would result in changing over 100 different places we use this method.